workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -478,10 +498,12 @@ for (name in names(res.list)){
   dds.i <- res.list[[name]][['dds']]
   res.i <- res.list[[name]][['res']]
   label <- res.list[[name]][['label']]
+  alpha.i <- res.list[[name]][['alpha']]
+  lfc.thresh.i <- res.list[[name]][['lfc.thresh']]
```

Comment:
if alpha isn't entered into the res.list pseudo-manually (see above comment), then here you can use  e.g.,`metadata(res.list[[name]][['res']])[['alpha']]` to figure it out